src/js/06-code.js

```diff
@@ -46,6 +46,8 @@ document.addEventListener('DOMContentLoaded', function () {
       input = window.neo4jDocs.copyableCommand(input)
     }
 
+    input = input.replace(/[ ]+\n/g, '\n').trimEnd()
```

Comment:
```suggestion
    input = input.replace(/[\s]+?\n/g, '\n').trimEnd()
```
Might be worth testing with this, in case somebody decided to use `TAB` to space annotations from code. A `?` might be needed to ensure `\s` doesn't eat the carriage return as well.

Reply:
If someone has used tab to space annotations from code we should change that in the asciidoc.

If any more testing is required for this fix I'm inclined to suggest we just update the one codeblock where we know a callout has caused a copy/paste problem.

Reply:
May I add some comments here?
* I find this callout feature really nice to explain complex code or shell commands blocks!
* right, it is not just curl what would cause errors, but at least all shell commands where a line break character ("\") is used before the callout
* The problem with shell commands and spaces at the end is the shell error message which doesn't help at all to understand the problem. And spaces are invisible :smile:. This would lead to user frustration.
I would like to use callouts so hope this will be fixed - Thx!

Reply:
If it helps get this out sooner, I would say we just focus on the copy button behavior and the use of a single space as that's the issue that has been identified in this specific case. Further work can be done in a separate PR.

Reply:
I think `[ |\t]` will work for us here if we want to also catch tabs. @stefano-ottolenghi please let me know if that misses anything obvious.

I've also added a listener for the copy event when the button either isn't used or isn't available.

Reply:
Looks good to me! There's only a commented-out line we should remember to remove, but otherwise good work! :cartwheeling:

Reply:
Thanks - I've deleted the line now